### ISSUE TITLE
Request: track path of key names in errors

### ISSUE THREAD

**Issue by braco** [litixsoft/lx-valid#2]
For example, this coffeescript code, which turns { foo: bar: "baz" } into ["foo", "foo.bar"]:

```
isObject = (obj) -> Object.prototype.toString.call(obj) is '[object Object]'
allProperties = (object, path=[]) ->
  props = []
  for key, val of object
    here = path.concat([key])
    props.push here.join(".")
    if isObject val
      props.push allProperties(val, here)...
  props
```

it would be handy if error messages had the full path of the error key, so error.property was "path.to.key" instead of just "key"

**Comment by braco** [comment]
Just realized these might need to go into flatiron/revalidator. True?

**Comment by 4kochi** [comment]
We released version 0.3.0 today which should handle your case. Please try it out.
